Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -8,33 +8,53 @@
 Installation on Google GKE
 **************************
 
+GKE Requirements
+================
+
+1. Install the Google Cloud SDK (``gcloud``) see [Installing Google Cloud SDK](https://cloud.google.com/sdk/install)
+
+2. Create a project
+
+::
+
+   export GKE_PROJECT=gke-clusters
+   gcloud projects create $GKE_PROJECT
+
+
+3. Enable the GKE API for the project
+
+::
+
+   gcloud services enable --project $GKE_PROJECT container.googleapis.com
+
 Create a GKE Cluster
 ====================
 
 You can apply any method to create a GKE cluster. The example given here is
-using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. 
+using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. This guide
+will create a cluster on zone ``europe-west4-a``, feel free to change the zone
```

Comment:
Why are you setting the zone? Most users will have a default zone. I'd rather add a step to the `GKE requirements` section than to make the commands longer. Does the command fail if the zone is not set?

Reply:
Because I didn't have a default zone as I was trying out this guide for the first time. The command did fail as the zone is required to be set.